pages/api/file/upload.ts

```diff
@@ -64,7 +64,7 @@ export default async function handle(req: NextApiRequest, res: NextApiResponse):
                     new Date(),
                 );
             } else if (user.role === roles.PARENT && path === accepted_type_paths[1]) {
-                await updateParentProofOfIncomeLink(session.user.email, file as string);
+                await updateParentProofOfIncomeLink(session.user.email, file as string, new Date());
```

Comment:
umm, something times me that this will be wack because of timezones.

Our deployment server might be in some other time zone. We want to add our times in UTC. Try this:
```suggestion
                await updateParentProofOfIncomeLink(session.user.email, file as string,  new Date(date.toUTCString()));
```

Reply:
@ZhangCreations Can I trouble you to do the same for Line 64: https://github.com/uwblueprint/social-diversity-for-children/pull/178/files#diff-b5edddd7380dfa1a232831d66755a08d15272b8a2959e7f64700e906c8b0b8d3L64
🙏